pom.xml

```diff
@@ -377,6 +378,14 @@
                 <version>1.0.0</version>
             </dependency>
             
+            <dependency>
+                <groupId>pl.edu.icm.spark-utils</groupId>
+                <artifactId>spark-utils</artifactId>
+                <version>0.0.1-SNAPSHOT</version>
+            </dependency>
```

Comment:
spark-utils define spark-core_2.10 and spark-sql_2.10 dependencies.
So now we will have spark libraries in runtime classpath and they will be packaged in production workflows.
Two solutions come to my mind. Don't know witch is better.
1) exclude spark on above dependency
2) mark spark scope as provided in spark-utils